src/Console/Command/FixCommand.php

```diff
@@ -184,7 +184,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         );
 
         $progressOutput = $showProgress && $stdErr
-            ? new ProcessOutput($stdErr, $this->eventDispatcher)
+            ? new ProcessOutput($stdErr, $this->eventDispatcher, count($resolver->getFinder()))
```

Comment:
This `count` is very expensive. It forces the iterator to go all the way through.